Gemfile

```diff
@@ -63,6 +63,7 @@ gem 'will_paginate-bootstrap'
 gem 'yt'
 gem 'yui-compressor'
 gem 'youtube_rails'
+gem 'google-api-client', '0.8'
```

Comment:
The upgrade doc explains it pretty clearly and I think @tansaku would agree it should be done so as not to use an outdated version. I can certainly help too.